modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-api/src/main/resources/content/Language.properties

```diff
@@ -52,6 +52,12 @@ override-type-mappings-help=Set custom mappings to fully replace the default map
 password-help=Set the password for authenticating to Elasticsearch if Authentication Enabled is checked.
 production-mode-enabled=Production Mode Enabled
 production-mode-enabled-help=Enable Production mode. If this is checked, Production mode will be enabled and the Operation Mode configuration will be ignored. Production mode connects to a remote standalone Elasticsearch cluster. If this is unchecked, the Operation Mode configuration will be used.
+proxy-host-help=Set the proxy host the client uses to connect.
+proxy-password=Proxy Password
+proxy-password-help=Set the proxy password to use if connecting through a proxy.
+proxy-port-help=Set the proxy port the client uses to connect.
+proxy-username=Proxy Username
+proxy-username-help=Set the proxy user name to use if connecting through a proxy.
```

Comment:
Recommend:

> Set the user name for connecting to the proxy.

Not sure I fully understand this so feel free to go with something else if I've made it inaccurate.